Tests/SwiftSyntaxBuilderTest/ExpressibleBuildablesTests.swift

```diff
@@ -0,0 +1,95 @@
+import XCTest
+import SwiftSyntax
+
+import SwiftSyntaxBuilder
+
+final class ExpressibleBuildablesTests: XCTestCase {
+    func testExpressibleAsMemberDeclListItem() {
+        let myStruct = StructDecl(identifier: "MyStruct", members: MemberDeclBlock(membersBuilder: {
+            VariableDecl(.var, name: "myFirstVar", type: "Int")
+
+            VariableDecl(letOrVarKeyword: .let, bindingsBuilder: {
+                PatternBinding(pattern: IdentifierPattern(identifier: .identifier("mySecondLet")),
+                               typeAnnotation: TypeAnnotation(type: SimpleTypeIdentifier("CGFloat")))
+            })
+
+            MemberDeclListItem(decl: VariableDecl(letOrVarKeyword: .let, bindingsBuilder: {
+                    PatternBinding(pattern: IdentifierPattern(identifier: .identifier("myOtherVar")),
+                                   typeAnnotation: TypeAnnotation(type: SimpleTypeIdentifier("String")))
+                })
+            )
+
+            StructDecl(identifier: "InnerStruct", members: MemberDeclBlock())
+        })
+        )
+
+        let syntax = myStruct.buildSyntax(format: Format())
+        XCTAssertEqual(syntax.description, """
+        struct MyStruct{
+            var myFirstVar: Int
+            let mySecondLet: CGFloat
+            let myOtherVar: String
+            struct InnerStruct{
+            }
+        }
+        """)
+    }
+
+    func testExpressibleAsCodeBlockItem() {
+        let myCodeBlock = SourceFile(eofToken: SyntaxFactory.makeToken(.eof, presence: .present)) {
```

Comment:
Not part of this PR but would it make sense to add 
```swift
extension TokenSyntax {
  static let eof = SyntaxFactory.makeToken(.eof, presence: .present)
}
```
So we could improve this to 
```suggestion
        let myCodeBlock = SourceFile(eofToken: .eof) {
```
And possibly try to remove the need to specify the `eofToken` in the `SourceFile` initializer altogether.

Reply:
Added in #314